### PR TITLE
implement not-implemented: GetInputContainerQueues

### PR DESCRIPTION
**Pull request by stishkin**

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/TaskOperations.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public interface ITaskOperations : IStatefulOrm<Task, TaskState> {
 
     IAsyncEnumerable<Task> SearchStates(Guid? jobId = null, IEnumerable<TaskState>? states = null);
 
-    IEnumerable<Container>? GetInputContainerQueues(TaskConfig config);
```
Comment (tevoinea) [inline src/ApiService/ApiService/onefuzzlib/TaskOperations.cs:18]: Did you mean to use `OneFuzzResult`? I don't think we use `Result<T, E>` anywhere outside of `FunctionalTests`

**Review by tevoinea — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/TaskOperations.cs (hunk 1 above)
Comment (Porges) [reply src/ApiService/ApiService/onefuzzlib/TaskOperations.cs:18]: We should probably just rename `OneFuzzResult` to `Result`…

Comment (stishkin) [reply src/ApiService/ApiService/onefuzzlib/TaskOperations.cs:18]: `OneFuzzResult` Error is of OneFuzz Error type `{int code, string[] errors}`

Anywhere in the code where `TaskConfigError` is returned it is returned as `ResultVoid<TaskConfigError>.Error(new TaskConfigError(err));`

In this specific case `ResultVoid` won't do

Next code iteration - maybe might make sense to get rid of `ResultVoid<TaskConfigError>` as well as fix up this code and use `OneFuzzResult` across the board

**Review by Porges — APPROVED** [review]

File: src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs [hunk 2]
```diff
@@ -84,7 +84,8 @@ public NotificationOperations(ILogTracer log, IOnefuzzContext context)
         // Nullability mismatch: We filter tuples where the containers are null
         return _context.TaskOperations.SearchStates(states: TaskStateHelper.AvailableStates)
             .Select(task => (task, _context.TaskOperations.GetInputContainerQueues(task.Config)))
-            .Where(taskTuple => taskTuple.Item2 != null)!;
+            .Where(taskTuple => taskTuple.Item2.IsOk && taskTuple.Item2.OkV != null)
+            .Select(x => (x.Item1, x.Item2.OkV))!;
```
Comment (Porges) [inline src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs:88]: We should return a named tuple here to make the code easier to understand.

**Pull request merged**